src/components/DropDownMenu/DropDownMenu.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import styles from './DropDownMenu.module.css';
+
+import { initialCities } from './constants.js';
+
+function DropDownMenu() {
+  const [searchValue, setSearchValue] = useState(null);
+
+  const getSearchedValue = e => setSearchValue(e.target.value);
+
+  const citiesToShow = searchValue
+    ? initialCities.filter(city => city.cityName.includes(searchValue))
+    : initialCities;
+
+  return (
+    <div className={styles.dropdownMenuContainer}>
+      <label>
+        Choose a city from this list:
```

Comment:
should this go into a contant?

Reply:
changes are addressed @virzhinia-petkova-redshelf

Reply:
did you decide against moving this to a constant or you forgot about it?

Reply:
Sorry, I didn't get your idea initially. Now all is clear. The labels have been moved in a separate file as discussed. @virzhinia-petkova-redshelf